### PR TITLE
Style treats

### PR DESCRIPTION
**Pull request by robertberry-zz**
norwich

### CODE REVIEW

**Comment by crifmulholland** [comment]
Beautiful!

File: static/src/stylesheets/module/facia/_treats.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            display: none;
+        }
+    }
+}
+
+.treats__treat {
+    @include fs-textSans(1);
+    @include button-height($base-size * 4);
+    @include border-radius(1000px);
+    @include border-box;
```
Comment (mattosborn) [inline static/src/stylesheets/module/facia/_treats.scss]: @include box-sizing(border-box);

File: static/src/stylesheets/_mixins.scss [hunk 2]
```diff
@@ -301,3 +301,9 @@
 
     @return $width;
 }
+
+@mixin border-box {
```
Comment (mattosborn) [inline static/src/stylesheets/_mixins.scss]: have a mixin for box-sizing already:
`@mixin box-sizing($box)`

File: static/src/stylesheets/module/facia/_treats.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        .has-page-skin & {
+            display: none;
+        }
+    }
+}
+
+.treats__treat {
+    @include fs-textSans(1);
+    @include button-height($base-size * 4);
+    @include border-radius(1000px);
```
Comment (mattosborn) [inline static/src/stylesheets/module/facia/_treats.scss]: `@include border-radius(50%);`

File: static/src/stylesheets/module/facia/_treats.scss [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        max-width: $left-column;
+    }
+
+    @include mq(wide) {
+        max-width: $left-column-wide;
+    }
+
+    @include button-colour(
+        #ffffff,
+        guss-colour(news-main-1, $pasteup-palette),
```
Comment (mattosborn) [inline static/src/stylesheets/module/facia/_treats.scss]: `colour(news-default);`

File: static/src/stylesheets/module/facia/_treats.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        #ffffff,
+        darken(guss-colour(neutral-3, $pasteup-palette), 10%),
```
Comment (mattosborn) [inline static/src/stylesheets/module/facia/_treats.scss]: `colour(neutral-3);`

File: static/src/stylesheets/module/facia/_treats.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        guss-colour(news-main-2, $pasteup-palette)
```
Comment (mattosborn) [inline static/src/stylesheets/module/facia/_treats.scss]: etc

File: static/src/stylesheets/module/facia/_treats.scss [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    );
+
+    &:hover,
+    &:focus,
+    &:active {
+        color: guss-colour(neutral-1, $pasteup-palette);
```
Comment (mattosborn) [inline static/src/stylesheets/module/facia/_treats.scss]: etc

**Comment by mattosborn** [comment]
:+1: aside from comments

**Comment by robertberry** [comment]
@mattosborn Ta!

**Pull request merged**